DIGITfiles/ParseSangerResults.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import os
+
+from Query import Query
+from Utils import *
+
+sangerQueriesAll = {
+    "A188v1": {},
+    "B73v5": {},
+    "W22v2": {}
+}
+
+
+####################################################################
+
+
+# sanger_blast_output/good_matches/
+def lookup_in_file(db, all_query_objs, flanking_file_name, list_name):
+    '''
+    Lookup query data in file corresponding to best genome for that query
+    '''
+    # print(flanking_file_name)
+    with open(
+            "order_" + flanking_file_name + "/blastResults/good_matches/" + db + "_vs_sanger_seq_" + flanking_file_name +
+            "_good_matches.txt", "r+") as parsed_file:
+        first_line = parsed_file.readline()
+        for line in parsed_file:
+            line_items = line.split(',')
+            line_items[-1].strip()
+            # print(list_name)
+            if line_items[0] in list_name:
+                query_data = line_items
+
+                col_headings = ("query acc.ver,subject acc.ver,percent identity"
+                                ",alignment length,mismatches,gap opens,q. start,q. end,s. sta"
+                                "rt,s. end,evalue,bit score,database,number of hits,percent di"
+                                "fference to next value,strand,q. start status,bit score statu"
+                                "s\n")
+                col_headings_list = col_headings.split(',')
+
+                query_obj = {
+                    col_headings_list[i]: query_data[i] for i in range(len(col_headings_list))}
+                # print(query_obj)
```

Comment:
Its worthwhile to take a couple of minutes to work out exactly what is happening in line 42. Instead, I asked ChatGPT:

![image](https://github.com/marilynel/DIGIT/assets/5107558/300ea241-668a-489d-9936-6973ab2619c7)